backend/src/main/resources/db/migration/V3__Task.sql

```diff
@@ -0,0 +1,55 @@
+CREATE TABLE task(
+    id INT AUTO_INCREMENT PRIMARY KEY,
+	name varchar(256),
+	description TEXT,
+    is_required BOOLEAN DEFAULT true,
+    milestone_id INT,
+
+    FOREIGN KEY (milestone_id) REFERENCES milestone(id)
+);
+
+INSERT INTO task (name, description, milestone_id)
```

Comment:
should the milestone_id be a part of the task itself or should there be some kind of connecting table that would be easier to change when the admin reassigns tasks?

Reply:
or does spring/mysql handle this on its own?

Reply:
Connecting tables are used for many-to-many relations or sometimes to represent optional relationships. From my understanding tasks can only belong to one milestone. I'm not sure why tasks would need to go in different milestones, but if that's the case we could add a connecting table, and also move the is required field there

Reply:
In that case, we can just put the field as NULL. I'm not sure if that requires any different annotations or anything but that is the simplest way